docs/src/modules/components/MarkdownLinks.js

```diff
@@ -18,11 +18,10 @@ export async function handleEvent(event, as) {
   event.preventDefault();
 
   let pathname = as.replace(/#(.*)$/, '');
-  // Add support for leading / in development mode.
+  // The leading / is only added to support static hosting (resolve /index.html).
+  // We remove it to normalize the pathname.
```

Comment:
I don't follow. Why do we let next add it and then remove it internally? That seems to make it unnecessarily complicated to reason about where pathnames point to.

Reply:
Confirmed, it's unnecessary.